srcpkgs/pulseeffects-legacy/template

```diff
@@ -4,18 +4,22 @@ version=4.8.5
 revision=1
 wrksrc="pulseeffects-${version}"
 build_style=meson
-hostmakedepends="itstool pkg-config gettext"
+hostmakedepends="itstool pkgconf gettext"
 makedepends="boost-devel glib-devel gsettings-desktop-schemas-devel
  gst-plugins-bad1-devel gtkmm-devel libebur128-devel lilv-devel
- pulseaudio-devel python3-gobject-devel sratom-devel
- libsndfile-devel"
-depends="calf gsettings-desktop-schemas gst-plugins-bad1
- gst-plugins-good1 pulseaudio python3-gobject python3-scipy"
+ pulseaudio-devel sratom-devel zita-convolver-devel
+ rubberband-devel libsamplerate-devel libsndfile-devel"
+depends="calf gst-plugins-good1"
 short_desc="Sound effects for systems using PulseAudio (legacy)"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="Artur Sinila <freesoftware@logarithmus.dev>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/wwmm/pulseeffects"
+changelog="https://github.com/wwmm/pulseeffects/blob/v${version}/CHANGELOG.md"
 distfiles="https://github.com/wwmm/pulseeffects/archive/v${version}.tar.gz"
 checksum=df1c4c4a9811c62a549822dacde3a9e36233ba3ec58817ae52a236f6181a507c
-python_version=3
 conflicts="pulseeffects"
+
+post_install() {
+	vdoc README.md
```

Comment:
Why do you think it should be installed? Which information does it provide?

Reply:
@paper42 
```
rg "vdoc README" srcpkgs | wc -l
739
```

Reply:
I don't generally see a problem with installing a readme file, some readmes provide useful quick start guides, but I don't see a reason in this case. The information inside is not useful imo for void users. There are some screenshots (which are broken btw), list of effects available (accessible from inside the application, so not useful imo), links to packages for distributions (useless) and other for us useless parts. The only part that could be interesting to users are the packages needed for plugins, but we already established that people can get that information from inside the application.

I think many of the packages which install a README don't have to, but I think that should be solved on a case by case basis when someone touches the package. If you ever update a package and notice it installs a readme that is not useful, I am sure void maintainers wouldn't mind if you removed it.

I am not a maintainer, so I might be wrong, but I don't see any value in installing this particular readme.

Reply:
OK, I see you point, I've already removed those `README`'s from this PR.